### ISSUE TITLE
"...: unary operator expected" error building on Windows 7

### ISSUE THREAD

**Issue by kenny-evitt** [LightTable/LightTable#2114]
I just fetched the latest *master* branch and ran *script/build.sh* from a Cygwin terminal window:

```
Retrieving lein-cljsbuild/cljs-compat/1.0.0-SNAPSHOT/cljs-compat-1.0.0-20151218.091126-41.pom from clojars
Retrieving lein-cljsbuild/cljs-compat/1.0.0-SNAPSHOT/cljs-compat-1.0.0-20151218.091126-41.jar from clojars
script/build.sh: line 17: [: /cygdrive/c/Program: unary operator expected
Please install npm before running this script.
```

I was able to get the script to work by changing line 17 to quote the `which` command, e.g. `` "`which npm`"`` instead of just `` `which npm` ``.

I'll submit a PR with this change in a few minutes.

**Comment by kenny-evitt** [comment]
PR #2115 fixes this for me.

**Comment by cldwalker** [comment]
Closed by #2115
